openrtb_ext/imp_richaudience.go

```diff
@@ -0,0 +1,9 @@
+package openrtb_ext
+
+type ExtImpRichaudience struct {
+	Pid         string  `json:"pid"`
+	SupplyType  string  `json:"supplyType"`
+	BidFloor    float64 `json:"bid_floor"`
+	BidFloorCur string  `json:"bid_floor_cur"`
```

Comment:
This would be great if you describe `BidFloor` and `BidFloorCur` in `static/bidder-params/richaudience.json` file

Reply:
in next commit i have solved: